lib/Http/Response/CodeInjector.php

```diff
@@ -118,20 +119,20 @@ private function injectIntoPresetSelector(string $html, string $code, string $se
 
     private function injectIntoDomSelector(string $html, string $code, string $selector, string $position, string $charset): string
     {
-        $dom = str_get_html($html);
+        $dom = new Crawler($html);
         if ($dom) {
```

Comment:
This `if` can go - if there is something wrong with the HTML contents, `new Crawler()` would throw an exception.